src/main/c/Posix/Makefile

```diff
@@ -11,17 +11,17 @@ COMPILE_OSX_X86             := o32-clang
 COMPILE_OSX_X86_64          := o64-clang
 COMPILE_OSX_ARM64           := oa64-clang
 COMPILE_ANDROID             := ndk-build
-COMPILE_FREEBSD_X86         := clang -target i386-unknown-freebsd11.2 --sysroot=$(HOME)/x-tools/i386-unknown-freebsd11.2
-COMPILE_FREEBSD_X86_64      := clang -target x86_64-unknown-freebsd11.2 --sysroot=$(HOME)/x-tools/x86_64-unknown-freebsd11.2
-COMPILE_FREEBSD_ARM64       := clang -target arm64-unknown-freebsd11.2 --sysroot=$(HOME)/x-tools/arm64-unknown-freebsd11.2
-COMPILE_OPENBSD_X86         := clang -target i386-unknown-openbsd6.2 --sysroot=$(HOME)/x-tools/i386-unknown-openbsd6.2
-COMPILE_OPENBSD_X86_64      := clang -target amd64-unknown-openbsd6.2 --sysroot=$(HOME)/x-tools/amd64-unknown-openbsd6.2
+COMPILE_FREEBSD_X86         := clang -target i386-unknown-freebsd12.4 --sysroot=$(HOME)/x-tools/i386-unknown-freebsd12.4
```

Comment:
(just a random person passing by)

looks like this PR is much more than just a CI spript? should it be a number of smaller PRs?